src/app/pcasts/controllers/search_episode_controller.py

```diff
@@ -0,0 +1,35 @@
+from . import *
+
+class SearchEpisodeController(AppDevController):
+
+  def get_path(self):
+    return '/search/episodes/<query>'
+
+  def get_methods(self):
+    return ['GET']
+
+  @authorize
+  def content(self, **kwargs):
+    search_title = request.view_args['query']
+    offset = request.args['offset']
+    max_search = request.args['max']
+    possible_episodes = episodes_dao.\
+        search_episode(search_title, offset, max_search)
+    cleaned_episodes = []
+    for e in possible_episodes:
```

Comment:
So you are matching the current apiary docs exactly, but I think we are trying to use the general schemas as much as possible to be consistent, so instead of doing all of this logic with `cleaned_episodes`, you could just return 
`return {'episodes': episode_schema.dump(possible_episodes).data}`

That's also why we are moving everything over to camel_case, since the serialization just takes the field names from the model